packages/lib/src/utils/date.ts

```diff
@@ -163,6 +163,12 @@ const formatDateTimeZone = (date) => {
   return d;
 };
 
+const formatDateTimeZoneTo9pm = (date) => {
```

Comment:
```suggestion
const setEndOfDay = (date: string): Date => {
```
ce n'est pas un formatage (passage de Date vers string), il faut nommer la méthode par rapport à ce qu'elle fait.
si on récupère une string, c'est `parseAndSetDateToEndOfDay`, sinon ça semblerai mieux de directement avoir une date dans les params

Reply:
OK bien compris. Oui le comportement souhaité est d'avoir une Date dans les params
J'ai été brain par le fonction formatDateTimeZone juste au dessus, qui ne fait rien du coup on est d'accord ? On formate une date en string mais on return la date initiale et non la string. Faudrait fix ca dans une autre PR je pense (pck elle est utilisé a plusieurs endroit)